packages/app-contracts/src/Deploy.tsx

```diff
@@ -248,9 +248,10 @@ class Deploy extends ContractModal<Props, State> {
   }
 
   private onSuccess = async (result: SubmittableResult) => {
-    const { history } = this.props;
+    const { api, history } = this.props;
```

Comment:
`api` is already being imported in line 14, that's why the build is failing. I don't know if it was an oversight or on purpose that `api` was not taken from the `props`, but if it's on purpose you can solve it by just renaming the import `import { api as apiUi, withApi, withMulti } from '@polkadot/ui-api';` and then changing these two lines to `apiUi`.

https://github.com/polkadot-js/apps/blob/7dbf43e108f08d017014895d8a6fb86b6ed4227d/packages/app-contracts/src/Deploy.tsx#L259

https://github.com/polkadot-js/apps/blob/7dbf43e108f08d017014895d8a6fb86b6ed4227d/packages/app-contracts/src/Deploy.tsx#L270

Reply:
I've implemented your suggestion, since I don't know whether it was on purpose or not. Is the `api` instance from props the same as the imported one?

Reply:
hmm, should be. Maybe @kwingram25 can clarify, he wrote the contracts app

Reply:
It is a copy gone wrong - it is used in the store since it doesn't have a context/props. https://github.com/polkadot-js/apps/pull/1347